testsuite/integration/src/test/xslt/enableJTS.xsl

```diff
@@ -21,7 +21,7 @@
     					  /*[local-name()='orb']">
         <xsl:copy>
             <xsl:attribute name="socket-binding"><xsl:value-of select="@socket-binding"/></xsl:attribute>
-            <xsl:attribute name="ssl-socket-binding"><xsl:value-of select="@ssl-socket-binding"/></xsl:attribute>
+            <!--<xsl:attribute name="ssl-socket-binding"><xsl:value-of select="@ssl-socket-binding"/></xsl:attribute>-->
```

Comment:
If this means we are disabling ssl, is this tracked anywhere so we know to reenable it?